docker/hisat2/Dockerfile

```diff
@@ -21,3 +39,14 @@ RUN \
 
 COPY  ./make_grch38_snp_tran_*.sh hisat2-2.1.0/
```

Comment:
Can you change these `COPY` tags to pull from the github repo to not rely on local objects? @rexwangcc Does this make sense to do for reliability, or do you think local copies are OK?

Reply:
Although the scripts are just in the same parent folder with this Dockerfile, I'd like to `COPY` them using git URL instead of copying them locally, for better reliability and versioning management.

Reply:
if the scripts are merged into master branch, we can do that way. but what if these scripts are from branch and the branch will be merged/removed later?

Reply:
I assume these situations should be really rare. If these happen often, when pulling and copying scripts from Github, we could use the hash code in the URL, instead of using the branch name. Github will always save all the committed code. 

For example, https://github.com/HumanCellAtlas/skylab/blob/2ce3c141696254cedec405c3f33a17a3c450d7ce/docker/python/Dockerfile this dockerfile refers to a removed file from a merged and deleted branch. 

However, I would avoid using hash code as much as possible, since it is not visually appealing and also semantic obscure.